server/client.py

```diff
@@ -155,3 +160,74 @@ def send_message_to(self, msg):
 
         # send msg
         ONLINE_USERS[to].send(PayloadMessage(msg.cmd, [self.user.name], msg.payload))
+
+    @register_cmd(CMD_CREATE_CHAT)
+    @login_required
+    def create_chat(self, msg):
+        """
+        Create new chat and send new chat id back to the client
+        """
+        chat_name = msg.payload
+        db_session = session()
+        chat = Chat(chat_name, owner=self.user, users=[self.user,])
+        db_session.add(chat)
+        db_session.commit()
+        log.info("New chat '{}' has been created".format(chat))
+        self.send(PayloadMessage(CMD_INFO, [], str(chat.id)))
```

Comment:
Partly my fault(I mean get_friends method), but I believe we shouldn't use PayloadMessage for these purposes. May be new InfoMessage class?